server/src/DBTablesMonitoring.cc

```diff
@@ -2540,6 +2540,23 @@ HatoholError DBTablesMonitoring::getNumberOfMonitoredItemsPerSecond
 	return HatoholError(HTERR_OK);
 }
 
+size_t DBTablesMonitoring::getNumberOfEvents(const EventsQueryOption &option)
+{
+	DBClientJoinBuilder builder(tableProfileEvents, &option);
+	string stmt =
```

Comment:
In this case the column name is determined at runtime, so stmt is also built at runtime.
I don't modify it.